ui/components/app/connected-accounts-permissions/connected-accounts-permissions.js

```diff
@@ -29,47 +44,64 @@ const ConnectedAccountsPermissions = ({ permissions }) => {
   );
 
   return (
-    <div className="connected-accounts-permissions">
-      <p
-        className="connected-accounts-permissions__header"
+    <Box className="connected-accounts-permissions" width={BlockSize.Full}>
+      <Box
+        display={Display.Flex}
+        flexDirection={FlexDirection.Row}
+        as="button"
         onClick={toggleExpanded}
+        width={BlockSize.Full}
+        justifyContent={JustifyContent.spaceBetween}
+        className="connected-accounts-permissions__header"
+        padding={0}
+        backgroundColor={BackgroundColor.backgroundDefault}
       >
-        <strong>{t('permissions')}</strong>
-        <button
-          className={classnames('fas', {
-            'fa-angle-down': !expanded,
-            'fa-angle-up': expanded,
-          })}
-          title={t('showPermissions')}
+        <Text
+          onClick={toggleExpanded}
+          as="h6"
+          variant={TextVariant.bodyMdMedium}
+        >
+          {t('permissions')}
+        </Text>
+
+        <ButtonIcon
+          size={ButtonIconSize.Sm}
+          iconName={expanded ? IconName.ArrowUp : IconName.ArrowDown}
+          ariaLabel={t('showPermissions')}
         />
-      </p>
-      <div
-        className={classnames(
-          'connected-accounts-permissions__list-container',
-          {
-            'connected-accounts-permissions__list-container--expanded':
-              expanded,
-          },
-        )}
-      >
-        <p>{t('authorizedPermissions')}:</p>
-        <ul className="connected-accounts-permissions__list">
-          {permissionLabels.map(({ label }, idx) => (
-            <li
-              key={`connected-permission-${idx}`}
-              className="connected-accounts-permissions__list-item"
-            >
-              <Checkbox
-                isChecked
-                isDisabled
-                id={`connected-permission-${idx}`}
-                label={label}
-              />
-            </li>
-          ))}
-        </ul>
-      </div>
-    </div>
+      </Box>
+      {expanded && (
```

Comment:
Seeing lots of `[condition] && (<component />)` here, when we generally prefer `[condition] ? <component /> : null`

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/20811/commits/e1739744968df5c9869c32187b166c2848c8dbf3